.github/changelog-config.json

```diff
@@ -0,0 +1,12 @@
+{
+    "categories": [
+        {"title": "## 🚀 Features", "labels": ["feature", "feat", "enhancement"]},
+        {"title": "## 🐛 Fixes", "labels": ["fix", "bug"]},
+        {"title": "## 🧪 Dependencies", "labels": ["dependency"]}
```

Comment:
```suggestion
        {"title": "## 🧪 Dependencies", "labels": ["type/update"]}
```

Reply:
> let's use the existing labels
> 
> what do you think about another category for type/refactor?



> let's use the existing labels
> 
> what do you think about another category for type/refactor?

Any reason to prefix everything with type/ ? doesn't make it redundant ? just the suffix make it clear it is a category or not? what do you think ?

Reply:
that's just what we have at the moment, @raminqaf created the category system